### PR TITLE
Fix/App invite

### PR DESCRIPTION
**Pull request by friedger**
This PR fixes a problem with not showing the app invite.

The app invite was lost when the user was transferred from the start activity to the main activity.

After:
device-2015-11-24-085733

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,4 @@
+sudo: false
```
Comment (tasomaniac) [inline .travis.yml]: This is nice but it makes the build fail in Travis. The failures may be because of this.

File: build.gradle [hunk 2]
```diff
@@ -6,10 +6,10 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:1.4.0-beta1'
+        classpath 'com.android.tools.build:gradle:1.5.0'
```
Comment (tasomaniac) [inline build.gradle]: Finally they are out of beta!

**Comment by tasomaniac** [comment]
Thank you for looking into this. Can you try to make the build pass and then I think it can be merged.

**Comment by friedger** [comment]
@tasomaniac build passed! :-)

**Pull request merged**